fetcher/amazon.go

```diff
@@ -83,6 +84,11 @@ func FetchUpdateInfoAmazonLinux2() (*UpdateInfo, error) {
 	return fetchUpdateInfoAmazonLinux(amazonLinux2MirrorListURI)
 }
 
+// FetchUpdateInfoAmazonLinux22 fetches a list of Amazon Linux2 updateinfo
```

Comment:
Is it `Amazon Linux22` instead of `Amazon Linux2`?

Reply:
done
i used 2022 instead os 22
Thanx